test/Conversion/TorchToLinalg/view.mlir

```diff
@@ -0,0 +1,39 @@
+// RUN: torch-mlir-opt <%s -convert-torch-to-linalg -split-input-file -verify-diagnostics | FileCheck %s
+
+// -----
+
+// CHECK-LABEL:   func.func @torch.aten.view$twotothree(
+// CHECK-SAME:  %[[ARG:.*]]: !torch.vtensor<[3,2],f32>) -> !torch.vtensor<[2,3],f32> {
+// CHECK:    %[[BUILTIN_TENSOR:.*]] = torch_c.to_builtin_tensor %[[ARG]] : !torch.vtensor<[3,2],f32> -> tensor<3x2xf32>
+// CHECK:    %[[CASTED:.*]] = tensor.cast %[[BUILTIN_TENSOR]] : tensor<3x2xf32> to tensor<3x2xf32>
+// CHECK:    %[[COLLAPSED:.*]] = tensor.collapse_shape %[[CASTED]] {{\[\[}}0, 1]] : tensor<3x2xf32> into tensor<6xf32>
+// CHECK:    %[[EXPANDED:.*]] = tensor.expand_shape %[[COLLAPSED]] {{\[\[}}0, 1]] : tensor<6xf32> into tensor<2x3xf32>
+// CHECK:    %[[EXPAND_CAST:.*]] = tensor.cast %[[EXPANDED]] : tensor<2x3xf32> to tensor<2x3xf32>
+// CHECK:    %[[BUILTIN_TENSOR_CAST:.*]] = torch_c.from_builtin_tensor %[[EXPAND_CAST]] : tensor<2x3xf32> -> !torch.vtensor<[2,3],f32>
+// CHECK:    return %[[BUILTIN_TENSOR_CAST]] : !torch.vtensor<[2,3],f32>
+
+func.func @torch.aten.view$twotothree(%ARG: !torch.vtensor<[3,2],f32>) -> !torch.vtensor<[2,3],f32> {
+    %I3 = torch.constant.int 3
+    %I2 = torch.constant.int 2
+    %LIST = torch.prim.ListConstruct %I2, %I3 : (!torch.int, !torch.int) -> !torch.list<int>
+    %VIEW = torch.aten.view %ARG, %LIST : !torch.vtensor<[3,2],f32>, !torch.list<int> -> !torch.vtensor<[2,3],f32>
+    return %VIEW : !torch.vtensor<[2,3],f32>
+  }
+
+// CHECK-LABEL: func.func @torch.aten.view$dynamictest(
+// CHECK-SAME:      %[[ARG:.*]]: !torch.vtensor<[?,?],f32>) -> !torch.vtensor<[?,?],f32> {
+// CHECK:        %[[BUILTIN_TENSOR:.*]] = torch_c.to_builtin_tensor %[[ARG]] : !torch.vtensor<[?,?],f32> -> tensor<?x?xf32>
+// CHECK:        %[[CASTED:.*]] = tensor.cast %[[BUILTIN_TENSOR]] : tensor<?x?xf32> to tensor<?x?xf32>
+// CHECK:        %[[BUILTIN_TENSOR_CAST:.*]] = torch_c.from_builtin_tensor %[[CASTED]] : tensor<?x?xf32> -> !torch.vtensor<[?,?],f32>
+// CHECK:        return %[[BUILTIN_TENSOR_CAST]] : !torch.vtensor<[?,?],f32>
+
+func.func @torch.aten.view$dynamictest(%ARG: !torch.vtensor<[?,?],f32>) -> !torch.vtensor<[?,?],f32> {
+    %C1 = torch.constant.int 1
+    %C0 = torch.constant.int 0
+    %LISTVALUE1 = torch.aten.size.int %ARG, %C0 : !torch.vtensor<[?,?],f32>, !torch.int -> !torch.int
```

Comment:
It is common in MLIR testing to just name these %0, %1 (run it through torch-mlir-opt to pretty print)

Reply:
That gives me an invalid variable error, that's why I changed them to those values

Reply:
Oh wait, you're talking about the ones outside the checks, okay, I'll change those back.